orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -111,6 +111,8 @@ def __init__(self, module):
             self._torch_alloc = self._torch_gpu_allocator.gpu_caching_allocator_raw_alloc_address()
             self._torch_free = self._torch_gpu_allocator.gpu_caching_allocator_raw_delete_address()
 
+        _utils._load_aten_op_executor_cpp_extension(self._loglevel < _logger.LogLevel.WARNING, self.is_rocm_pytorch)
```

Comment:
Also, how would this work when we have multiple-ortmodules? 
Will we compile the same extension code multiple times?

Reply:
OK